airbyte-cdk/python/airbyte_cdk/sources/file_based/file_based_source.py

```diff
@@ -85,7 +85,7 @@ def streams(self, config: Mapping[str, Any]) -> List[AbstractFileBasedStream]:
                 stream_config = FileBasedStreamConfig(**stream)
                 streams.append(
                     DefaultFileBasedStream(
-                        config=FileBasedStreamConfig(**stream),
+                        config=stream_config,
```

Comment:
we're alreadying parsing into `FileBasedStreamConfig` above, so no need to do it twice